bundle/next/eclipse-che-preview-openshift/manifests/che-operator.clusterserviceversion.yaml

```diff
@@ -268,14 +215,15 @@ spec:
       identityProviderRealm: 'myrealm'
       identityProviderClientId: 'myClient'
       ```
-    #### Self-signed Certificates and TLS Secrets
 
-    Eclipse Che uses self-signed certificates by default, as TLS mode is on by default.
+    #### Self-signed Certificates
```

Comment:
The title is wrong. We do not know which kind of certs are default for the cluster.